employee-rostering-gwtui/src/main/java/org/optaweb/employeerostering/gwtui/client/viewport/availabilityroster/AvailabilityRosterPageViewport.java

```diff
@@ -52,12 +51,16 @@
 import org.optaweb.employeerostering.shared.spot.Spot;
 
 @Templated
-public class AvailabilityRosterPageViewport extends DateTimeViewport<AvailabilityRosterView, AvailabilityRosterMetadata> implements IsElement {
+public class AvailabilityRosterPageViewport extends DateTimeViewport<AvailabilityRosterView, AvailabilityRosterMetadata>
+        implements
+        IsElement {
 
     @Inject
     private AvailabilityRosterPageViewportBuilder viewportBuilder;
     @Inject
     private ManagedInstance<AvailabilityGridObject> employeeAvailabilityGridObjectInstances;
+    @Inject
+    private DateTimeUtils dateTimeUtils;
```

Comment:
It's strange to see a Utils being injected, as normally classes that are called Utils have only static methods (and we don't call those from instances). We might want to rename DateTimeUtils.

Reply:
`DateTimeUtils` uses Errai's `TranslationService`, which needs to be injected for formatting reasons. All Util classes in optaweb-employee-rostering-gwtui `util`'s package are injected, although `CommonUtils` and `PromiseUtils`can be converted to static methods since they don't have any injects.

Reply:
Hmm, I see your point. It's not something I'd except with a class called Utils, but it's still a utils class in nature. Ok, let's leave it as is.